vlib/db/pg/README.md

```diff
@@ -51,3 +51,11 @@ Read this section to learn how to install and connect to PostgreSQL
 [*Windows*](https://www.postgresqltutorial.com/install-postgresql);
 [*Linux*](https://www.postgresqltutorial.com/postgresql-getting-started/install-postgresql-linux);
 [*macOS*](https://www.postgresqltutorial.com/postgresql-getting-started/install-postgresql-macos).
+
+## Using Parameterized Queries
+
+Parameterized queries (exec_param, exec_param2, exec_param_many) in V require the use of the following syntax: ($1) ($2)...($n). The number following the $ specifies which parameter from the argument array to use.
+```v
```

Comment:
Use 
\`\`\`v ignore
here, since the sample code is not complete enough to compile on its own.

If you do not, then `v check-md vlib/db/pg/README.md` will fail, and that is run on the CI.